docs/DeveloperGuide.md

```diff
@@ -227,7 +250,7 @@ the filtered transaction with a success message.
     - 3a1. UniCa$h displays no transactions, and informs the user that all transactions have been retrieved. 
     - Use Case ends.
 
-**Use Case: UC05 - Finding a Transaction**
+**Use Case: UC06 - Finding a Transaction**
```

Comment:
Believe there might be some issues with merging with master now since I recently relabelled some of the use case ids at the bottom, could you merge to verify, thanks